docker-compose.yml

```diff
@@ -3,7 +3,8 @@ version: '3'
 services:
 
   mysql:
-    image: mysql:5
+    image: mysql:8
+    command: '--default-authentication-plugin=mysql_native_password'
```

Comment:
What is the purpose of this argument? Is it needed because of the version bump from 5 to 8?